core/metacling/src/TClingCallbacks.cxx

```diff
@@ -362,6 +362,16 @@ bool TClingCallbacks::LookupObject(const DeclContext* DC, DeclarationName Name)
    if (Name.getNameKind() != DeclarationName::Identifier)
       return false;
 
+   Sema &SemaR = m_Interpreter->getSema();
+   auto *D = dyn_cast<Decl>(DC);
+   if (D) {
+      SourceLocation Loc = D->getLocation();
+      if (Loc.isValid() && SemaR.getSourceManager().isInSystemHeader(Loc)) {
+         // We will not help the system headers, sorry.
```

Comment:
Could expand on the comment to explain why we don't want to "help the system headers" ? :)

Reply:
It's a copy from the other function...

Reply:
oh well :(, not very clear either way :)

Reply:
I extended both comments :wink: